llvm/include/llvm/ADT/DenseMapInfo.h

```diff
@@ -151,7 +151,8 @@ template<> struct DenseMapInfo<unsigned long long> {
   static inline unsigned long long getTombstoneKey() { return ~0ULL - 1ULL; }
 
   static unsigned getHashValue(const unsigned long long& Val) {
-    return (unsigned)(Val * 37ULL);
+    return DenseMapInfo<unsigned>::getHashValue(Val) ^
```

Comment:
the easy way is `detail::combineHashValue`

Reply:
+1 to that

Reply:
Done